2] loto/main.py

```diff
@@ -0,0 +1,45 @@
+import random
+
+
+def get_user_numbers():
+    user_numbers = set()
+    while len(user_numbers) < 6:
+        try:
```

Comment:
ten try muze byt klidne mensim podobe jako to mas v tom prvnim prikladu. Neni to chyba ale je to lepsi